app/services/admin/participants/search.rb

```diff
@@ -32,6 +32,9 @@ def search_conditions
         .or(TeacherProfile.where(id: search_term))
         .or(User.where(id: search_term))
         .or(ParticipantIdentity.where(external_identifier: search_term))
+        .or(School.where(name: search_term))
```

Comment:
@peteryates This isnt quite working as expected mainly for testing it. Need to chat to you about how the model works for the user -> school as ive seen changes that we are moving more towards the profile. 

Ao if you have a user who is an mentor as example how is best to load the school records. This actually doesnt work as planned as the user can have two schools i found with testing although this might just be how i set up the tests.